.github/workflows/test.yaml

```diff
@@ -0,0 +1,46 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [windows-2019, windows-latest]
+    
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    # Environment requirements:
+    # https://learn.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment
+    - name: Test docker
+      run: |
+        docker version
+        docker pull hello-world
+        docker run hello-world
+
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Install Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: 'stable'
```

Comment:
Shouldn't be necessary, it should already be installed as of https://github.com/actions/runner-images/blob/main/images/windows/Windows2019-Readme.md#language-and-runtime

Reply:
I removed this from a follow-up PR, and ended-up having issues with both windows versions:
https://github.com/rancher/wins/actions/runs/8094456813/job/22119187899?pr=159
https://github.com/rancher/wins/actions/runs/8094456813/job/22119187595?pr=159